Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.10 as base
+FROM ubuntu:20.04 as base
```

Comment:
why downgrade?

Reply:
It's the last `LTS`. I don't understand why but 20.10 seemd to be called Groovy Gorilla and already reached the [end of life](https://wiki.ubuntu.com/Releases).

I guess we can also use a lighter image than Ubuntu, but we'll try in a separate PR.

Reply:
Feel free to merge :)